documentation/modules/operators/con-deleting-managed-topics.adoc

```diff
@@ -54,6 +54,34 @@ If the deletion fails, it is shown in the status of the resource.
 
 When the finalization tasks are successfully executed, the finalizer is removed from the metadata, and the resource is fully deleted.  
 
-Finalizers also prevent related resources from being deleted. 
-If the unidirectional Topic Operator is not running, it won't be able to remove the `metadata.finalizer`. 
-Consequently, an attempt to delete the namespace that contains the `KafkaTopic` resource won't complete until either the operator is restarted, or the finalizer is otherwise removed (for example using `kubectl edit`). 
\ No newline at end of file
+Finalizers also serve to prevent related resources from being deleted. 
+If the unidirectional Topic Operator is not running, it won't be able to remove its finalizer from the `metadata.finalizers`. 
+And any attempt to delete the `KafkaTopic` resources directly or by deleting the namespace will result in failure or timeout. 
```

Comment:
I think the namespace deletion being "stuck in a terminating state" will be a common pitfall. Do we want to mention it more clearly?